packages/css-data/tsconfig.dts.json

```diff
@@ -2,6 +2,7 @@
   "extends": "./tsconfig.json",
   "include": ["src"],
   "compilerOptions": {
+    "rootDir": "src",
```

Comment:
why, having that it worked without before

Reply:
typescript is weird 🤷

Reply:
@TrySound I am moving it to tsconfig/base.json so that its set for everything, is this alright?

Reply:
didn't work, but now I removed it and it seems working

Reply:
Paths should be never in reusable config. Typescript compute paths relatively to config file.

Reply:
already put the change to the local config